G/GMT/build_tarballs.jl

```diff
@@ -76,7 +76,7 @@ dependencies = [
     Dependency(PackageSpec(name="LibCURL_jll", uuid="deac9b47-8bc7-5906-a0fe-35ac56dc84c0"); compat="7.73.0, 8.0.1")
     Dependency("NetCDF_jll", compat="400.902.210")
     Dependency("PROJ_jll", compat="901.300.0")
-    Dependency(PackageSpec(name="GDAL_jll", uuid="a7073274-a066-55f0-b90d-d619367d196c"); compat="301.900")
+    Dependency(PackageSpec(name="GDAL_jll", uuid="a7073274-a066-55f0-b90d-d619367d196c"); compat="302.1000")
```

Comment:
Changing compat bounds means that the version number itself has to change (it's a limitation of current implementation of package manager/registry).

Reply:
OK, thanks. I think I did that (increased from 6.5.2 to 6.5.3)